mage_ai/frontend/oracle/icons/custom/Convert.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+import { BaseIconProps } from '../BaseIcon';
+import { DEFAULT_SIZE } from '../shared/constants';
+
+export type LogoWithNameProps = {
```

Comment:
This isn't used. Remove.

Reply:
Sorry, was left behind from me trying to set up a custom icon.